easybuild/tools/config.py

```diff
@@ -590,9 +590,27 @@ def update_build_option(key, value):
     """
     # BuildOptions() is a (singleton) frozen dict, so this is less straightforward that it seems...
     build_options = BuildOptions()
+    orig_value = build_options._FrozenDict__dict[key]
     build_options._FrozenDict__dict[key] = value
     _log.warning("Build option '%s' was updated to: %s", key, build_option(key))
 
+    # Return original value, so it can be restored later if needed
+    return orig_value
+
+
+def update_build_options(key_value_dict):
+    """
+Update build options as specified by the given dictionary (where keys are assumed to be build option names).
+Returns dictionary with original values for the updated build options.
```

Comment:
Please indent to match the `"""` line above

Reply:
Haha, hey, I committed _your_ suggested change here! ;-) OK ok... will do :)